snapcraft.yaml

```diff
@@ -1014,6 +1014,7 @@ parts:
     source-type: git
     plugin: rust
     build-packages:
+      - libtirpc-dev
```

Comment:
Presumably that dependency was implicit when we used `core22`, right? If so, having this in a separated commit would make it easier to backport to other branches. Thanks

Reply:
Thanks! Fixed.